core/services/commander/main.py

```diff
@@ -34,6 +35,25 @@ class ShutdownType(str, Enum):
     POWEROFF = "poweroff"
 
 
+def run_command(command: str) -> subprocess.CompletedProcess:
+    user = "pi"
+    password = "raspberry"
```

Comment:
It would be good to have this somewhere on the installation script and import the config for both from some fixed place.

Reply:
We are already doing the same thing for filebrowser, we can fix the user/login configuration in a future PR.

Reply:
k